app/assets/javascripts/tylium.js

```diff
@@ -34,6 +34,7 @@
 //= require tylium/keyboard_shortcuts
 
 //= require tylium/modules/auto_save
+//= require tylium/modules/server_auto_save
```

Comment:
is it worth doing a `/auto_save/server` and `/auto_save/local`? I think @sean-yeoh's work is adding a `/local_auto_save`

Reply:
I like the idea.

Reply:
Created `/auto_save` and moved the related server auto save js to `/auto_save/server.js`